src/forms/labels/input-label.js

```diff
@@ -112,7 +110,7 @@ function InputLabel ({
       }
       {
         tooltip &&
-        <span className="tooltip-trigger" onClick={ toggleTooltipShown }/>
+        <span className="tooltip-trigger" onClick={() => toggleTooltipShown()}/>
```

Comment:
Same question about the redefinition of `onClick` as above.

Reply:
Answered here https://github.com/LaunchPadLab/lp-components/pull/562#discussion_r910332521!